src/test/scala/is/hail/methods/PCRelateSuite.scala

```diff
@@ -207,4 +207,18 @@ class PCRelateSuite extends SparkSuite {
     assert(mapSameElements(actual, truth, compareDoubleQuartuplets((x, y) => math.abs(x - y) < 1e-2)))
   }
 
+  @Test
+  def kinshipFiltering() {
+    val vds = hc.importVCF("src/test/resources/sample.vcf.bgz")
+
+    val pcs = SamplePCA.justScores(vds.coalesce(10), 2)
+
+    val truth = PCRelateReferenceImplementation(vds, pcs, maf=0.01)._1
+      .filter { case (_, (kin, _, _, _)) => kin >= 0.125 }
+    val actual = runPcRelateHail(vds, pcs, 0.01, 0.125)
+
+    assert(truth.size > 0)
+    assert(mapSameElements(actual, truth, compareDoubleQuartuplets((x, y) => math.abs(x - y) < 1e-2)))
```

Comment:
FYI, `quartuplet` is not in the (miriam-webster) dictionary. Consider changing to `quadruplet`.

Reply:
Addressed in https://github.com/hail-is/hail/pull/2271